recipes-bsp/u-boot/u-boot-iot2050.inc

```diff
@@ -20,7 +20,7 @@ U_BOOT_CONFIG = "iot2050_defconfig"
 U_BOOT_BIN = "flash.bin"
 
 SPI_FLASH_IMG = "${U_BOOT_BIN}"
-SPI_FLASH_DEPLOY_IMG = "iot2050-image-boot.bin"
+SPI_FLASH_DEPLOY_IMG ??= "iot2050-image-boot.bin"
```

Comment:
Does this make any sense? Nothing is producing that image. I would just drop this line.

Reply:
I think `u-boot-iot2050_202x.xx-upstream.bb`  may still need it. Or just delete this upstream bb?

Reply:
In fact, upstream needs to be converted as well in order to still work on both variants... Good point.

Reply:
That could be going to another PR... not on the critial path.

There are some new upstreaming patches anyway, the second stage for this PR, maybe this upstream bb could be addressed in that stage.

Reply:
OK, then break the upstream recipe again for now.